openpype/hosts/unreal/integration/UE_5.0/Source/OpenPype/Private/OpenPypePublishInstance.cpp

```diff
@@ -2,107 +2,151 @@
 
 #include "OpenPypePublishInstance.h"
 #include "AssetRegistryModule.h"
+#include "AssetToolsModule.h"
+#include "NotificationManager.h"
```

Comment:
for some reason it doesn't work for me but this does:
```suggestion
#include "Framework/Notifications/NotificationManager.h"
```